packages/ag-charts-community/src/dom/focusIndicator.ts

```diff
@@ -23,9 +23,18 @@ export class FocusIndicator {
         this.swapChain.addListener('swap', (parent) => this.onSwap(parent));
     }
 
-    updateBounds(bounds: Path | BBoxValues | undefined, hoverRect?: { x: number; y: number }) {
+    updateBounds(bounds: Path | BBox | undefined, hoverRect?: BBox) {
         if (bounds === undefined) {
-            // skip
+            return;
+        }
+
+        const boundsBBox = bounds instanceof Path ? bounds.getBBox() : bounds;
+        const boundsVisible = hoverRect != null && boundsBBox != null ? hoverRect.intersects(boundsBBox) : true;
+
+        if (!boundsVisible) {
```

Comment:
This condition is unnecessary. The `relativeClip` in the `else` clause will set the height or width to 0 if necessary.

The only advantage of this `if(!boundsVisible)` condition is that it also handle SVG-based focus indicators which currently has no clipping in production. But I would prefer to leave the SVG rendering logic unchanged, even if our current implementation is a bit buggy.